src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -171,12 +171,15 @@ public PodTemplate(PodTemplate from) {
         this.setServiceAccount(from.getServiceAccount());
         this.setSlaveConnectTimeout(from.getSlaveConnectTimeout());
         this.setActiveDeadlineSeconds(from.getActiveDeadlineSeconds());
+        this.setIdleMinutes(from.getIdleMinutes());
         this.setVolumes(from.getVolumes());
         this.setWorkspaceVolume(from.getWorkspaceVolume());
         this.yaml = from.yaml;
         this.setYamls(from.getYamls());
-        this.setShowRawYaml(from.isShowRawYaml());
-        this.setNodeProperties(from.getNodeProperties());
+        this.showRawYaml = from.showRawYaml;
```

Comment:
Just to avoid having the field be defined in the copy when null in the original.